src/layouts/hass-subpage.ts

```diff
@@ -80,6 +90,10 @@ class HassSubpage extends LitElement {
         border-bottom: var(--app-header-border-bottom, none);
         box-sizing: border-box;
       }
+      .toolbar a {
+        color: var(--sidebar-text-color);
```

Comment:
Shouldnt this be?
```suggestion
        color: var(--app-header-text-color);
```

Reply:
Lifted it from somewhere else where it was used like this so needs to be adjusted there as well.

Reply:
https://github.com/home-assistant/frontend/pull/10711